homepage 4-10 content

karigaron17 opened the pull request:
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #190

Test URLs:
- Before: https://main--maidenform--hlxsites.hlx.live/
- After: https://190-create-homepage-content-4-10-layout--maidenform--hlxsites.hlx.live/

herzog31 commented on styles/styles.css:
```diff
@@ -90,118 +173,6 @@
 
-/* fallback fonts for Zarah font family */
-
-@font-face {
```
Is there a reason you removed the fallback fonts? Font files should be loaded delayed via `lazy-styles.css` so they don't block the initial page rendering. If you need to add additional fallback fonts, you can leverage https://github.com/pixel-point/fontpie.

When using fonts in your styles, please use the variables we defined, e.g. `font-family: var(--ff-zahrah-semibold)`.

karigaron17 replied:
The fallback was to Arial, which is a san-serif font and not at all like the Zahrah font. We can just use the font-family: font, serif for a fall back for now. We be shifting all the fonts soon and removing both Gilroy and Zahrah all together.

herzog31 replied:
Good point. We can adjust the fallback fonts to use a serif font like Times for Zahrah. To optimize performance, we should (for now) load font files only in `lazy-styles.css` however and do it in a way it doesn't cause layout shift. That's why we use the linked tool to generate a fallback font that creates the same dimensions as Zahrah.

> The font variable did not seem to carry over and work in the fragment file either.
We should check this. The variables are defined globally, so they should be available everywhere on the page.

karigaron17 replied:
I am working on reverting this back to the lazy-styles and a fallback with a serif font. the only place the variable didn't seem to work was in the fragment html <style> tag file. I will try it again with the lazy-style updated.

herzog31 commented on fragments/2023_04/2023-04-10.html:
```diff
@@ -0,0 +1,917 @@
+  /* hero ----------------------------------------------------------*/
+  @media (max-width: 599px) {
+    .mdf-hp-hero {
+      background: url('//cdn.maidenform.com/wysiwyg/MDF/Franklin/2023_04_10/040623_MF_HP_MB_Hero_03b.jpg?trim=620,0,0,0') 0 85%/ 100% 53% no-repeat;
```
Please ensure all images are delivered either via Franklin directly or served from the same host (franklin.maidenform.com). I recommend to upload the images to Sharepoint and publish them via the Sidekick.

herzog31 commented on fragments/2023_04/2023-04-10.html:
```diff
@@ -0,0 +1,917 @@
+</style>
+
+<div class="homepage-wrapper">
+  <!-- Top Banner -->
```
All of these are very good candidates for blocks btw.

karigaron17 replied:
We will slowly start moving the repeatable parts to blocks as we can working with the designers. So much of our content is one time layout specific and not the same layout again.

karigaron17 commented:
@herzog31 I believe the fonts and image comments should all be resolved now. I updated the Zahrah fallback fonts and added them back in and the fonts are loading correct now. Moved the images to SharePoint and updated all the src urls to for them.

I will look at splitting a few elements out to blocks for the next build, but will keep some of the css part of the html fragment as the color will need to be custom per update.

looks like there were other updates to the repo that when synced are showing as errors in the PR now.

herzog31 commented on styles/styles.css:
```diff
@@ -223,24 +224,24 @@
-  --ff-zahrah-regular-italic: 'Zahrah Regular Italic 400', 'Zahrah Regular Italic 400 Fallback', serif;
-  --ff-zahrah-light-italic: 'Zahrah Light Italic 300', 'Zahrah Light Italic 300 Fallback', serif;
-  --ff-zahrah-light: 'Zahrah Light 300', 'Zahrah Light 300 Fallback', serif;
+  --ff-zahrah-bold: 'Zahrah Bold', serif;
```
```suggestion
  --ff-zahrah-bold: 'Zahrah Bold', 'Zahrah Bold Fallback', serif;
```
You forgot about all fallback fonts in the Zahrah font variables.

herzog31 reviewed (approved):
Looks good to merge 👍
